File: APIflow/php/tutorials/simpleMsg.php
<?php

include('../APIFlow.php');

if(count($argv) > 0){
	$sessionKey = auth($argv[1], $argv[2], $argv[3]);	
}else if(count($_GET) > 0){
	$sessionKey = auth($_GET["email"], $_GET["password"], $_GET["apiKey"]);	
}

$processId = startProcessWithMsg($sessionKey, "simpleMsg");
$process = getProcess($sessionKey, $processId);
foreach($process["TASKS"] as $t){
	$res = completeTask($sessionKey, $t["ID"]);
	print_r($res);
}

?>

File: APIflow/ruby/tutorials/simpleMsg.rb
require_relative '../APIFlow.rb'

email, password, apiKey = ARGV[0], ARGV[1], ARGV[2]

flow = APIFlow.new(email, password, apiKey)

sessionKey = flow.sessionKey()

#we've already setup simple.bpmn20.xml on staging
processId = flow.startProcessWithMsg(sessionKey, "simpleMsg")
process = flow.getProcess(sessionKey, processId)

#complete tasks with empty json body
for t in process["TASKS"]
	res = flow.completeTask(sessionKey, t["ID"])
	print(res)
end